### PR TITLE
#4621 create/delete volume on/from specify path

### PR DESCRIPTION
**Pull request by chia7712**
### Checklist

- [x] [Review Contribution Guidelines](https://oharastream.github.io/en/docs/master/contributing/)
- [x] Add at least one reviewer (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add an assignee (of course, you are the assignee by default)
- [x] Do not set milestone, projects and labels (PR labels are added automatically by Jenkins)
- [x] Link to an issue
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (If the failed tests are unrelated to this PR, add a comment in the PR thread)

### CODE REVIEW

File: ohara-configurator/src/test/scala/oharastream/ohara/configurator/TestConfiguratorBuilder.scala [hunk 1]
```diff
@@ -100,13 +99,6 @@ class TestConfiguratorBuilder extends OharaTest {
       .serviceCollie(Mockito.mock(classOf[ServiceCollie]))
       .build()
 
-  @Test
-  def reassignK8sClient(): Unit =
```
Comment (chia7712) [inline ohara-configurator/src/test/scala/oharastream/ohara/configurator/TestConfiguratorBuilder.scala:104]: we don't expose ```k8sClient``` anymore so this test is useless

File: ohara-configurator/src/test/scala/oharastream/ohara/configurator/TestConfiguratorBuilder.scala [hunk 2]
```diff
@@ -137,40 +129,6 @@ class TestConfiguratorBuilder extends OharaTest {
     } finally apiServer.close()
   }
 
-  @Test
-  def testK8SClientNamespaceNone(): Unit = {
```
Comment (chia7712) [inline ohara-configurator/src/test/scala/oharastream/ohara/configurator/TestConfiguratorBuilder.scala:141]: we have unit test for k8s client so this test is duplicate

File: ohara-configurator/src/test/scala/oharastream/ohara/configurator/TestConfiguratorBuilder.scala [hunk 3]
```diff
@@ -137,40 +129,6 @@ class TestConfiguratorBuilder extends OharaTest {
...
-      val configurator: Configurator = Configurator.builder.k8sClient(k8sClient).build()
-      Await
-        .result(configurator.containerClient.log(podName, None), Duration(10, TimeUnit.SECONDS))
-        .head
-        ._2 shouldBe logMessage
-    } finally apiServer.close()
-  }
-
-  @Test
-  def testK8SClientNamespace(): Unit = {
```
Comment (chia7712) [inline ohara-configurator/src/test/scala/oharastream/ohara/configurator/TestConfiguratorBuilder.scala:157]: ditto

**Comment by jackyoh** [comment]
retry collieIT

**Comment by chia7712** [comment]
retry

**Comment by chia7712** [comment]
retry connector

**Comment by chia7712** [comment]
retry collieIT

**Pull request merged**